guide/volume-3-landing-page.txt

```diff
@@ -0,0 +1 @@
+Volume III details the necessary steps to building a communications strategy to be used for engaging the public with the BRT project, educating customers on how to use the system, and marketing the project to a wide array of potential users. This volume is crucial to illuminating the benefits of BRT and establishing the distinction between BRT and less effective forms of transit. Strategies can be crafted to address the concerns of specific stakeholders through a multitude of media formats, and these strategies� success should be measured in order to further increase their effectiveness (Chapter 9). Outreach is also essential to spreading the word about BRT, but it is also essential to engage the public through participatory meetings and workshops to get buy-in (Chapter 10). Once the public is actively providing feedback to officials on how the BRT system can best work for them, the public is then empowered to take ownership of the system and see it is as their transport system. Marketing and branding (Chapter 11) is how the BRT project team can communicate that community ownership and unique identity of a system through its name, logo, color scheme, wayfinding, and customer service, all of which ensure that the system is user-friendly, attractive, simple, and easy to use.
```

Comment:
This file is causing the build failure because, unexpectedly, there is a reserved control character from a non-ASCII/non-UTF-8 set (`hex 0092 => �`, just after `media formats, and these strategies`).  If you remove it, things should work.

First, since that isn't a used control character (is a character that's being reserved for the past 20 years for future control use), there's no reason why it should be here.  Probably the text editor used is at fault.  Which was it?

Second, there's the issue on the encoding.[1]  Since we can't stick to pure ASCII – essentially, US-only letters, numbers and a handful of symbols that don't even cover the basic dash types (-, –, —) or non-code quote marks (“,”) – but at the same time the guide has to be readable and writable from anywhere in the world, we need to use a Unicode[2]-aware encoding; of those, UTF-8 is the most appropriate for general file storage, PDFs, mobile devices and the Web in general.

The difficulty is: while there are a ton of possible encoding for any file (and a single Windows+Office installation will just by default use 4 of them, depending of the application being run), there's no way to pass that information along with the file, much less to securely infer the encoding from the file contents.  And it get's worse: from those 4 default Windows+Office encodings, one of them can change from computer to computer, depending of the language set on Windows and other settings.

Things are easier on Mac and Linux, since both default to what the Web and most servers have already agreed: to use UTF-8, an encoding that supports all scripts and is reasonably space-efficient.

---

[1] The encoding is how applications translate our text from and into bytes.
[2] Unicode is a huge effort to categorize and assign unique number to every used glyph or character in the world.  Currently it supports every live script (despite the being constant discussion and improvement) and quite a few dead ones to.